src/components/EventDetails/EventDetails.js

```diff
@@ -841,6 +843,32 @@ export class EventDetail extends React.Component<Props, State> {
           ],
         };
         break;
+      case SMART_WALLET_ACCOUNT_DEVICE_ADDED:
+        eventData = {
+          name: 'New Smart Wallet account device',
+          actionTitle: 'Added',
+          buttons: [
+            {
+              title: 'View on the blockchain',
```

Comment:
Just a note to expect changes:
Links to check trx on blockchain will be moved to event time as per #2021